changelog.d/20241112_220331_pjhinton_sc_37389_trasfer_client_collection_search_filter_entity_type.rst

```diff
@@ -0,0 +1,4 @@
+Added
+~~~~~
+
+- Add `filter_entity_type` keyword argument on `TransferClient.endpoint_search()`. (:pr:`NUMBER`)
```

Comment:
Sorry for not catching this on my first review pass. RST requires double-backticks for inline-code, and reserves single backticks for some of its special syntaxes (like sphinx roles).

```suggestion
- Add ``filter_entity_type`` keyword argument on ``TransferClient.endpoint_search()``. (:pr:`NUMBER`)
```

Reply:
@sirosen wrote:

> Sorry for not catching this on my first review pass. RST requires double-backticks for inline-code, and reserves single backticks for some of its special syntaxes (like sphinx roles).

Thanks for your patience as I cleanup up beginner errors! Addressed in commit f8052c9440caf22b2b2bfd3278809b76c10f2578.